sample_code/ep210/README.md

```diff
@@ -0,0 +1,49 @@
+# [github actions vulnerability or "why bug bounties are a scam" (intermediate)](https://youtu.be/_fpXyS_i1xE)
+
+Today I talk about a vulnerability I found in github actions involving the `pull_request_target` feature and how it escalates to credential access / full repository access.  I found over ~350 vulnerable repositories including ones owned by google, amazon, microsoft, alibabi, psf and more and document my experience (or lack thereof) with bug bounty programs.
+
+## Interactive examples
+
+### Bash
+
+```bash
+git clone git@github.com:<username>/gha-test
+# git clone https://github.com/<username>/gha-test
```

Comment:
This repo (in your `asottile` account) doesn't exist anymore, so I used a placeholder.

Reply:
yeah I did it on another account: https://github.com/throwaway-gha/gha-test

Reply:
Yes, but this is the target repository in your video. And the command itself was for cloning a fork of the target repository - which is used to create the PRs.
If you want, I can substitute the placeholder with the `throwaway-gha` username?